src/Aspire.Hosting/Postgres/PgAdminConfigWriterHook.cs

```diff
@@ -20,9 +22,10 @@ public Task AfterEndpointsAllocatedAsync(DistributedApplicationModel appModel, C
 
         using var stream = new FileStream(serverFileMount.Source!, FileMode.Create);
         using var writer = new Utf8JsonWriter(stream);
-
         var serverIndex = 1;
```

Comment:
(super nit) Can you move this to right above the for loop? It makes it hard to read when variables are defined in random places.